server/graphql/common/expenses.ts

```diff
@@ -510,7 +510,7 @@ type ExpenseData = {
   payeeLocation?: Record<string, unknown>;
   items?: Record<string, unknown>[];
   attachedFiles?: Record<string, unknown>[];
-  collective?: Record<string, unknown>;
+  collective?: Record<string, any>;
```

Comment:
Any reason for doing this change? Using `any` is usually discouraged and generates an ESLint warning. If it's about accessing properties, you can usually solve it by using brackets (eg. `settings?.['disablePublicExpenseSubmission']`) for accessing the property.

Reply:
@Betree : I did this change to bypass the typescript CI error we are getting; 

`server/graphql/common/expenses.ts:536:42 - error TS2339: Property 'disablePublicExpenseSubmission' does not exist on type 'unknown'.`

Let me think of an alternative approach. 🤔

Reply:
Try with brackets, as mentioned above that should work: 

```es6
settings?.['disablePublicExpenseSubmission']
```

Reply:
@Betree : Ops sorry, I somehow didn't try that. Yes it works, thanks. 👍🏽